packages/modal/src/ModalContainer.js

```diff
@@ -8,6 +8,7 @@
 import PropTypes from 'prop-types';
 
 import { useModal } from './useModal';
+import { FocusJailContainer } from '@zendeskgarden/container-focusjail/src';
```

Comment:
Rather than import the full container for the correct `PropTypes`, let's just copy over the PropType values. This will match what we do elsewhere.

```jsx
focusOnMount: PropTypes.bool,
environment: PropTypes.object,
```